README.md

```diff
@@ -1,15 +1,22 @@
 # Ploigos GitHub Workflows
 
-In order to use this workflow for your application, you must do the following -
+This repository houses the Ploigos workflows defined within a Github Actions [reusable workflow](https://docs.github.com/en/actions/using-workflows/reusing-workflows). For ease of use, there are templates that can be used within the examples directory. Please replace all values that begin with `REPLACE_`.
```

Comment:
This needs a bit more context. We should talk about the purpose of this repo at the top, like "This is a reusable GitHub Actions workflow. You can reference it in your application" (can be paraphrased)

Reply:
I think that's what the first sentence states. Maybe more is needed? If so, we may want to mob it.